Rpourzand cost behavioral change faq (#8626)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Cost optimizations/job-frequency-change.md

---

> title:  How will cost and reduction estimates be affected if I increase the frequency of my jobs?

Reviewer: Hiya @luna-bianca 

The title is slightly long (approx 15 words), it might be worth shortening it to something like **How does increasing job frequency affect cost and reduction estimates?**, **How does job frequency affect cost and reduction estimates?** (suggestions)?

> title:  How will cost and reduction estimates be affected if I increase the frequency of my jobs?

Author: Hey @nataliefiann , how does this look:
```suggestion
title:  How does increasing job frequency affect cost reduction estimates?
```

I did not use "Job frequency and cost estimates" to maintain consistency (FAQ page titles are in the form of a question) -- let me know if this title works!

Reviewer: Hiya @luna-bianca 

I like this and I think it follows the style of the FAQs.

Kind Regards
Natalie

> title:  How will cost and reduction estimations change if I update my jobs to run them more frequently?

Author: ```suggestion
title:  How will cost and reduction estimates be affected if I increase the frequency of my jobs?
```

description: "Explanation of how increasing run frequency affects cost reductions"
sidebar_label: 'Job frequency and cost reductions'

> id: job-frequency-change

Reviewer: I might be being too finicky / picky. Why does the URL have the word "change" in it (job-frequency-change)? Should it capture some info from the title like job-frequency, job-frequency-cost-reductions

Author: Changed the filename here: https://github.com/dbt-labs/docs.getdbt.com/pull/8626/commits/485902004bdba130a9e581bee8f067644f1e04d0

---

Cost reduction metrics reflect how dbt optimizes compute costs by reusing existing results instead of running the same model again.

When you increase your job run frequency (for example, because performance improvements make it easier to schedule jobs more often), dbt has more opportunities to reuse models. As reuse increases, dbt optimizes more compute, which means your reported cost reductions may also increase.

> This metric shows the efficiency impact of reuse within your current workload. It reflects how dbt optimizes compute through intelligent execution to maximize warehouse value, not just reduce total warehouse spend.

Reviewer: ```suggestion
This metric shows the efficiency impact of model reuse within your current workload. It reflects how dbt optimizes compute through intelligent execution to maximize warehouse value, not just reduce total warehouse spend.
```

> This metric shows the efficiency impact of reuse within your current workload. It reflects how dbt optimizes compute through intelligent execution to maximize warehouse value, not just reduce total warehouse spend.

Reviewer: ```suggestion
This metric shows the efficiency impact of reuse within your current workload. It measures compute avoided through reuse, rather than overall reductions in total warehouse spend.
```

Reviewer: Hiya @luna-bianca 

This is a non-blocking suggestion. 

Kind Regards
Natalie

Reviewer: This phrase / these words "intelligent execution to maximize warehouse value" were slightly vague to me. But could just be me

Hence the non-block suggestion
